packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx

```diff
@@ -34,7 +34,7 @@ export default SliceSimulatorPage;`,
 
 const steps = [
   InstallSliceSimulator({
-    code: `npm install --save @prismicio/react @prismicio/slice-simulator-react`,
+    code: `npm install --save @prismicio/react @prismicio/slice-simulator-react @prismicio/client@^6 @prismicio/helpers`,
```

Comment:
in the init script we don't precise the client version : 

```
${PRISMIC_REACT_PACKAGE_NAME} ${PRISMIC_CLIENT} ${SLICE_SIMULATOR_REACT} ${PRISMIC_HELPERS}
```

So what is the good version ?

Reply:
latest,
I versioned this one as we ran into issues when @pavery007 was updating a project :)

Reply:
ok so maybe we should also update the client installation in the init also right ?

Reply:
probably not.

Adding latest is for when they already have v5 installed, so in an old project that they are updating.